docs/rules/require-jsdoc.md

```diff
@@ -87,6 +93,13 @@ function foo() {
 /**
  * It returns 10
  */
+var foo = () => {
```

Comment:
Could you provide an example with `@props and @returns` just to make it clear that full jsDoc syntax is supported.